spack/packages/dla-future/package.py

```diff
@@ -70,8 +71,8 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("pika@0.15.1:", when="@0.1")
     depends_on("pika@0.16:", when="@0.2.0")
     depends_on("pika@0.17:", when="@0.2.1")
-    depends_on("pika@0.18:", when="@0.3.0:")
-    depends_on("pika@0.19.1:", when="@master")
+    depends_on("pika@0.18:", when="@0.3")
+    depends_on("pika@0.19.1:", when="@0.4.0:")
```

Comment:
```suggestion
    depends_on("pika@0.19.1:", when="@0.4:")
```
?

Reply:
Not sure. IMO this is safer in case it will happen to have two pika spec for different 0.4 versions (as for 0.2.x).